code/processes/gateway.q

```diff
@@ -131,7 +130,7 @@ canberun:{
  // check if it's possible to run anything
  availServers:availableserverstable[1b];
  if[0=count avail:distinct exec servertype from availServers;
-	:update required:(),available:(),handles:() from 0#0!queryqueue];
+  :update required:(),available:(),handles:() from 0#0!queryqueue];
```

Comment:
New line for closing square to make it more obvious where this condition ends. I think adding an extra space indentation will make this more obvious.

Reply:
@ThomasSmyth Would it be possible to codify your code review guidelines as a `qlint` script?

Reply:
@jzabroski development of a linter is currently under consideration.